javascripts/main.js

```diff
@@ -22,14 +22,14 @@ function getRadioValue()
 
```

Comment:
this file isn't quite right.  you will have some problems with it later down the road.  Leave your original IIFE in here and take out everything else and put it in a new file.  Add that script tag for the new file to your index.html AFTER all other script tags